pkg/systemlogmonitor/logwatchers/filelog/log_watcher.go

```diff
@@ -121,7 +121,7 @@ func (s *filelogWatcher) watchLoop() {
 		}
 		log, err := s.translator.translate(strings.TrimSuffix(line, "\n"))
 		if err != nil {
-			klog.Warningf("Unable to parse line: %q, %v", line, err)
+			klog.V(5).Infof("Unable to parse line: %q, %v", line, err)
```

Comment:
There are 2 types of errors: timestamp parsing error and message filtering error. This hides both errors. But I think you only want to hide the message filtering error, right? https://github.com/kubernetes/node-problem-detector/blob/master/pkg/systemlogmonitor/logwatchers/filelog/translator.go#L59

I think in the original design, we only considered the pattern where the message regular expression is common in the file log. This is true for kernel log for example. Can you share more of the use case here? It's probably valid and we may want to log here instead of erroring out: https://github.com/kubernetes/node-problem-detector/blob/master/pkg/systemlogmonitor/logwatchers/filelog/translator.go#L74

Reply:
> Can you share more of the use case here? 

I'll update the introductory comment on the PR with the use cases where this became a problem.

Reply:
See also #1038